Finjector.Web/ClientApp/src/components/Details/DetailsAeErrors.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert } from "reactstrap";
+
+interface DetailsAeErrorsProps {
+  errors: string[] | undefined;
+  hasWarnings: boolean | undefined;
+  warnings: string[] | undefined;
+}
+
+const DetailsAeErrors = (props: DetailsAeErrorsProps) => {
+  const { errors, hasWarnings, warnings } = props;
+
+  if (!errors && !warnings) return null;
+
+  return (
+    <div>
+      {!!errors &&
+        errors.length > 0 &&
+        errors.map((error, i) => {
```

Comment:
I think you can just do `{errors?.map(error=>.....}` and it'll work just as well.  if errors is null/undefined or has no elements you'll get the same result with a lot less ceremony